mailserver/mailserver.go

```diff
@@ -126,7 +127,7 @@ func (s *WMailServer) Init(shh *whisper.Whisper, config *params.WhisperConfig) e
 	s.w = shh
 	s.pow = config.MinimumPoW
 
-	if err := s.setupRequestMessageDecryptor(config); err != nil {
+	if err = s.setupRequestMessageDecryptor(config); err != nil {
```

Comment:
What was wrong with the previous version?

Reply:
It was shadowing the previous declaration. But I guess its fine to keep it